main.go

```diff
@@ -59,6 +62,29 @@ func main() {
 	// Initialize the database handler
 	pizzaOven := database.NewPizzaOvenDbHandler(databaseHost, databasePort, databaseUser, databasePwd, databaseDbName)
 
+	// Initializes configuration using a provided yaml file
+	config := &server.Config{NeverEvictRepos: make(map[string]int)}
+	var configParser struct {
+		NeverEvictRepos []string `yaml:"never-evict-repos"`
+	}
+
+	if configPath != "" {
+		configFile, err := os.ReadFile(configPath)
+		if err != nil {
+			sugarLogger.Fatalf("Could not read yaml configuration file: %s", err.Error())
+		}
+
+		err = yaml.Unmarshal(configFile, &configParser)
+		if err != nil {
+			sugarLogger.Fatalf("Could not unmarshal configuration file: %s", err.Error())
+		}
+
+		for _, repo := range configParser.NeverEvictRepos {
+			config.NeverEvictRepos[repo] = 0
```

Comment:
Do you think this hashmap makes more sense as a `map[string]bool` where it's a simple true / false instead of a int?

Reply:
Yep, memory wise just incurring a byte with bool type instead of int is better. I think the int could be flexible for collecting cache hit metrics of the neverEvictRepos, but it makes more sense to just have a separate flow for metrics within Get/Put.

Reply:
Hmm interesting. Collecting hit metrics could be an interesting thing to surface in the future. Not sure what that would look like. But I suppose for now, bool is great.